### PR TITLE
Bump Jenkins image tag from 2.249.1 to 3.2.0-rc0-2.249.1

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: charts/ks-devops/charts/jenkins/values.yaml [hunk 1]
```diff
@@ -54,6 +54,7 @@ Master:
     -XX:+DisableExplicitGC
     -XX:+UnlockDiagnosticVMOptions
     -XX:+UnlockExperimentalVMOptions
+    -Dhudson.security.csrf.DefaultCrumbIssuer.EXCLUDE_SESSION_ID=true
```
Comment (LinuxSuRen) [inline charts/ks-devops/charts/jenkins/values.yaml:57]: You can get more details about this from https://www.jenkins.io/doc/upgrade-guide/2.176/

**Comment by JohnNiang** [comment]
LGTM

**Comment by LinuxSuRen** [comment]
hi @JohnNiang , please help to check the new commit https://github.com/kubesphere-sigs/ks-devops-helm-chart/pull/6/commits/205b3ca2e32284cee98d7d89e11125df8055db98

It's worth to change the version of Jenkins chart due to we changed the image tag.

**Comment by JohnNiang** [comment]
> hi @JohnNiang , please help to check the new commit [205b3ca](https://github.com/kubesphere-sigs/ks-devops-helm-chart/commit/205b3ca2e32284cee98d7d89e11125df8055db98)
> 
> It's worth to change the version of Jenkins chart due to we changed the image tag.

It's more reasonable. LGTM

**Pull request merged**